build.sh

```
@@ -0,0 +1,29 @@
+export CFLAGS="${CFLAGS}"
+export CXXFLAGS="${CXXFLAGS}"
+export LDFLAGS="${LDFLAGS}"
+
+mkdir -p build
+cd build
+
+cmake .. \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  -DENABLE_FUZZING=ON
+
+make -j$(nproc)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)